cmd/docker/main.go

```diff
@@ -76,7 +78,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err := vicbackends.Init(cli.portLayerAddr, productName, &vchConfig); err != nil {
+	if err := vicbackends.Init(cli.portLayerAddr, productName, &vchConfig, cli.insecureRegistries); err != nil {
```

Comment:
It's not getting pulled from the command line.  cli is just a structure we use for command line flags, but the insecure registry flag is pulled from the vchconfig.